examples/remote-ui/app/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="https://fav.farm/🛰️" />
+    <link rel="stylesheet" href="./style.css" />
+
+    <title>Getting started • Remote DOM</title>
```

Comment:
If you are going to keep the examples separate, this should probably have a more specific title.